debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -162,6 +162,18 @@
                                                    .withDependents(COLLECTION_LIST_NAME)
                                                    .withDescription("The databases for which changes are to be captured");
 
+        /**
+     * A comma-separated list of regular expressions that match the fully-qualified namespaces of collections to be monitored.
```

Comment:
This comment seems like a Copy & Paste glitch, I don't think it should mention collections here.